### DISCUSSION TITLE
Watching and Listening type status overwriting the Playing type status

### DISCUSSION THREAD

**Discussion by ghost** [Rapptz/discord.py#6031] [Q&A] [answered]
This is the list of all the status
![image](https://user-images.githubusercontent.com/60844449/98776387-4459b480-2408-11eb-9e20-8a99075587d5.png)

![image](https://user-images.githubusercontent.com/60844449/98776666-d792ea00-2408-11eb-9462-5c9578437dc0.png)
If the status changes to watching or listening type status, it the type never changes.

Suppose, if status changes to **Watching PYTHON BOT Server**
Then the status remains as **Watching Fortnite**, but it is supposed to be **Playing Fortnite**

Also, there are no errors showing up in the terminal.

Thanks in Advance 😊

**Comment by Rapptz** [comment, marked as answer]
A few issues.

1. You're using `commands.Cog.listener()` on something that isn't a listener. This isn't an error but it is a bit odd to see.
2. The `or` keyword does not work like this. You want to do `status in ('a', 'b')` instead.

**Reply by ghost** [reply]
Hey @Rapptz , Thank you so much. It works.
